recipes/dask-glm/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "dask-glm" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "5a38d17538558fe6a3457cd67eed0a90a5dff51a9eaebb496efb68fc432ed89a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
```

Comment:
The `requirements.txt` specifies version restrictions as well as `cloudpickle`:

https://github.com/dask/dask-glm/blob/master/requirements.txt

any reason to not include them here?

Reply:
Also is `setuptools` a `run` requirement since `pkg_resources` is used:

https://github.com/dask/dask-glm/blob/master/dask_glm/__init__.py

While it's not causing an error, then `__version__` wouldn't be defined if it's missing.

Reply:
Cloudpickle is an indirect dependency via distributed, but isn't used anywhere directly. I'll remove that line from the requirements in dask-glm.

I've added the lower bounds, along with setuptools.